$OpenBSD: patch-configure_ac,v 1.10 2019/11/07 15:32:56 ajacoutot Exp $

Fix build with newer glib2mm (see include/glibmm-2.4/glibmm/timeval.h).
extension/timer.cpp:49:16: error: no member named 'assign_current_time' in 'Glib::TimeVal'
    expiration.assign_current_time();

Index: configure.ac
--- configure.ac.orig
+++ configure.ac
@@ -881,7 +881,7 @@ if test "x$enable_strict_build" != "xno"; then
 	
 	# Ensure that no deprecated glibmm symbols are introduced.
 	# lp:inkscape builds cleanly with this option at r10957
-	CPPFLAGS="-DGLIBMM_DISABLE_DEPRECATED $CPPFLAGS"
+	# CPPFLAGS="-DGLIBMM_DISABLE_DEPRECATED $CPPFLAGS"
 	
 	dnl Pango 1.32.4 uses a deprecated Glib symbol:
 	dnl   https://bugzilla.gnome.org/show_bug.cgi?id=689843
